opsdroid/connector/slack/__init__.py

```diff
@@ -70,85 +63,65 @@ def __init__(self, config, opsdroid=None):
             await self.disconnect()
             raise
 
-    async def reconnect(self, delay=None):
-        """Reconnect to the websocket."""
-        try:
-            self.reconnecting = True
-            if delay is not None:
-                await asyncio.sleep(delay)
-            await self.connect()
-        finally:
-            self.reconnecting = False
-
     async def disconnect(self):
         """Disconnect from Slack."""
-        await self.slacker.close()
+        self.listening = False
```

Comment:
Shouldn't these two lines really be the other way around :D

Reply:
Probably :)